README.md

```diff
@@ -11,19 +11,18 @@ This extension helps to use [Ansible](https://www.ansible.com/) efficiently in V
   - Auto completion. Auto completion Ansible directives, modules and plugins from Ansible doc.
   - Code snippets.  Press `Ctrl + Space`, Ansible playbook code snippets will show up.
   - Syntax highlighting.
-  - Run playbook from Terminal.
-    - On Windows, run Ansible inside docker.
-    - On Non-windows platform, provide option to run Ansible from docker or from local Ansible installation.
+  - Run playbook from Docker.
+  - Run playbook from local Ansible installation.
   - Run playbook from [Cloud Shell](https://azure.microsoft.com/en-us/features/cloud-shell/). 
 
 
 ## Requirements
 
 |platform|prerequisite|
 |--------|-----------|
-|All| nodejs >= 6.0 |
-|Windows/Other platforms choose docker|docker|
-|Platforms choose local| Ansible |
+|Platforms choose Cloud Shell| nodejs >= 6.0 |
+|Platforms choose Docker|Docker|
```

Comment:
What means "Platforms choose Docker" etc?

Reply:
refined the wording